package.json

```diff
@@ -13,12 +14,14 @@
 		"node": ">=8"
 	},
 	"scripts": {
-		"test": "xo && ava && tsd"
+		"build": "del dist && tsc",
+		"test": "npm run build && xo source/** && nyc ava",
```

Comment:
Use the XO `files` config instead of specifying it here.

Reply:
From what I read and tested, the `files` is only used in the `overrides` option. I don't want a override here, I just want the linting to run in the source folder.

Since there's no `includes` or `cwd` in `xo`, I see 2 options:
1. Ignore all folders except `source` with the `ignores` option.
2. Leave this part as it is.

Do you have any other suggestion about this?

Reply:
Ah, right. Ok, let's go with 2 then.